corehq/form_processor/models.py

```diff
@@ -530,6 +534,10 @@ def doc_type(self):
         return dt
 
     @property
+    def get_id(self):
```

Comment:
don't love that we have yet another way to get the id, but this makes the code a lot cleaner for repeaters